maintenance/projects/js-toolkit/packages/generator-liferay-js/src/target-angular-portlet/templates/src/app/dynamic.loader.ts.ejs

```diff
@@ -7,16 +7,19 @@ import {
 } from '@angular/core';
 
 import LiferayParams from '../types/LiferayParams'
+import { AppComponent } from './app.component';
```

Comment:
Sure you want the whitespace inside the curlies? I know Prettier isn't touching this file, but if it was, it would remove the whitespace.

Reply:
Prettier add that space (in my vscode) :sweat_smile: .

Reply:
Leaving the spaces as it's the formatting we are using in the other source files (see [this one](https://github.com/izaera/liferay-frontend-projects/blob/a12566f69d7364e2ff18f6297f2051049ddba8e5/maintenance/projects/js-toolkit/packages/generator-liferay-js/src/target-angular-portlet/templates/src/app/app.module.ts.ejs) that hasn't been touched by this PR for instance).

Reply:
Yeah, our Prettier has been [told not to look at theses files](https://github.com/izaera/liferay-frontend-projects/blob/a12566f69d7364e2ff18f6297f2051049ddba8e5/.prettierignore#L5). I just called it out because I figured you might have made the change by hand.